addons/mrp/tests/test_bom.py

```diff
@@ -883,10 +885,12 @@ def test_bom_report_planning_with_producible_qty(self):
         self.assertEqual(report_values['lines']['operations_time'], 30.0)
         self.assertEqual(report_values['lines']['producible_qty'], 0)
 
-        # The planning is limited to 700 days, fill the entire planning and keep 15 minutes
+        # The planning is limited to 4 weeks, fill the entire planning and keep 15 minutes
         # available, so that we can still plan a single operation.
         calendar = workcenter.resource_calendar_id
-        date_to = fields.Datetime.today() + timedelta(days=699)
+        date_to = fields.Datetime.today() + timedelta(days=14 * 2 - 1)
```

Comment:
Rather than complexifying the test more, I think we could move the `full_availability()` helper method from `TestWorkOrderProcess` to `TestMrpCommon` and simply set all days as 24h working days

This would also simply `end_of_day`, as it would be `date_start + timedelta(days=1)`.

What do you think?